### DISCUSSION TITLE
Retrieving PayOuts returning 404 error

### DISCUSSION THREAD

**Discussion by Clarry100** [hendt/ebay-api#184] [Q&A] [answered]
My Python code routine looks to me like it conforms to the ebay API doco. But when run it is returning a 404 error. 
Previously getting the auth code and access_token successfully. Scope=transactions_r
Here is the routine. Any ideas please?

 def get_bank_transactions(access_token, Start_Date, End_Date):

    if DEBUG: print("[get_bank_transactions] activated")

    # Base endpoint
    bank_url = "https://api.ebay.com/sell/finances/v1/payout"
    limit = 100
    offset = 0
    date_range = format_ebay_date_range(Start_Date, End_Date) # get the start & end date/time in the ISO format. Note: UTC times

    headers = {
        "Authorization": f"Bearer {access_token}",
        "Content-Type": "application/json",
        "X-EBAY-C-MARKETPLACE-ID": "EBAY_AU"
    }
    params = {
        "limit": limit,
        "offset": offset,
        "filter": f"payoutDate:[{date_range}]",
        "filter": f"payoutStatus:[SUCCEEDED]"
    }

    response = requests.get(bank_url, headers=headers, params=params)
    if DEBUG: print("[get_bank_transactions] Request URL sent:= ", response.request.url)    

    if response.status_code != 200:
        print(f"[get_bank_transactions] Error {response.status_code}: {response.text}")
        return []
    
    # Parse and return payouts
    return response.json().get("payouts", [])

**Comment by dantio** [comment, marked as answer]
Hey @Clarry100 this is the eBay node API not the python one :)

**Comment by Clarry100** [comment]
Yes thanks. I found out that I had the wrong url.
Should have been
 "https://apiz.ebay.com/sell/finances/v1/payout"
I've changed that and the 404 has gone but I'm getting a 500 error.
Says its internal server issue and to request help from ebay.
So waiting to hear back from  them.

**Comment by Clarry100** [comment]
I also solved the 500 errors.
It was the fact I had 2x filters set up. System apparently does not like that. So I edited the params to look like this:

params = {
    "limit": limit,
    "offset": offset,
    "filter": f"payoutDate:[{date_range}]"
}
Works fine now.
